torchrl/objectives/ppo.py

```diff
@@ -708,7 +708,7 @@ def forward(self, tensordict: TensorDictBase) -> TensorDict:
         td_out = TensorDict(
             {
                 "loss_objective": -neg_loss.mean(),
-                "kl": kl_val.detach().mean(),
+                "kl_to_prev": kl_val.detach().mean(),
```

Comment:
why is not starting with "loss_*"?

Reply:
there's a detach just after

Reply:
true, we could separate that loss too
in this case i did not do it for consistency with clip
such that one can safely do 
```
loss = losses["loss_actor"] + loss["loss_critic"]
```
otherwise the KL module needs one more term.

Reply:
If we want to change that behaviour it has to be in a separate PR bc it is bc breaking

Reply:
also for context non-loss values are metadata
we should probably ppt them in a separate nested td